code/main.py

```
@@ -0,0 +1,122 @@
+from tensorflow.keras.models import Model
+from tensorflow.keras.layers import Dense, Input, Embedding, Flatten, Concatenate
+from tensorflow.keras.utils import to_categorical
+import matplotlib.pyplot as plt
+
+# Step 1: Load and Merge Datasets
+def load_data():
+    print("Loading data...")
+    requirements_df = pd.read_csv("jobRequirements.csv")
+    job_data_df = pd.read_csv("jobData.csv")
+    merged_df = pd.merge(job_data_df, requirements_df, on="jobID", how="inner")
+    return merged_df
```

Ruff (0.8.2)

Unnecessary assignment to `merged_df` before `return` statement

Remove unnecessary assignment

(RET504)